### PR TITLE
Plotting binary updates

### PR DESCRIPTION
**Pull request by egthomas**
This pull request makes a few behind-the-scenes changes to the plotting binaries, including

- consolidation of `plot_time` functions from several different plotting binaries into the `binplotlib` library
- removal of old / unused `FrameBufferClip` code from `fov_plot` (this dates back to pre-3.0 versions of RST)
- addition of include guards for the various `binplotlib` header files

All plotting binaries should function the same on this  branch as on `develop`.

### CODE REVIEW

File: codebase/superdarn/src.lib/tk/binplotlib.1.0/include/plot_time.h [hunk 1]
```diff
@@ -24,11 +24,34 @@ along with this program. If not, see <https://www.gnu.org/licenses/>.
 Modifications:
 */
```
Comment (ecbland) [inline codebase/superdarn/src.lib/tk/binplotlib.1.0/include/plot_time.h:25]:
```suggestion
Modifications:
    E.G. Thomas (2022-07): consolidated plot_time functions from several different plotting binaries
*/
```

File: codebase/superdarn/src.lib/tk/binplotlib.1.0/src/plot_time.c [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+Modifications:
+*/
```
Comment (ecbland) [inline codebase/superdarn/src.lib/tk/binplotlib.1.0/src/plot_time.c:26]:
```suggestion
Modifications:
    E.G. Thomas (2022-07): consolidated plot_time functions from several different plotting binaries
*/
```

**Review by ecbland — COMMENTED** [review]
This looks good @egthomas. I've suggested some modification lines for the files that were changed substantially. Then I'm happy to merge!

**Comment by egthomas** [comment]
> This looks good @egthomas. I've suggested some modification lines for the files that were changed substantially. Then I'm happy to merge!

Thanks @ecbland!

**Pull request merged**